rusty_pizza_server/src/order_model/meals.rs

```diff
@@ -31,11 +31,33 @@ impl Meals {
         self.meals.insert(id, meal);
         self.meals.get_mut(&id).unwrap()
     }
+
+    pub fn set_paid(&mut self, paid: Money) {
+        self.paid = paid;
+    }
+
+    pub fn set_tip(&mut self, tip: Money) {
+        self.tip = tip;
+    }
+
+    pub fn calculate_total_price(&self) -> Money {
+        let mut total_price = Money::new(0, 0);
+        for (_, meal) in self.meals.iter() {
```

Comment:
If only the values are used then iteration should go over `.values()`:
```suggestion
        for meal in self.meals.values() {
```
(see below before implementing this)

Reply:
I haven´t seen this in the docs
https://doc.rust-lang.org/stable/rust-by-example/flow_control/for.html
Works, so I will use `values`

Reply:
It's documented [here](https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.values).